### PR TITLE
JS3 Week2 homework added

### PR DESCRIPTION
**Pull request by KavyaReddy732**

### CODE REVIEW

File: javascript/javascript3/week2/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
+// Movies API//
+async function movie() {
```
Comment (bashirmja) [inline javascript/javascript3/week2/index.js:2]: Functions should name with a verb and a name. 
for example: getData(), SetDate() and ets

File: javascript/javascript3/week2/index.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    console.log(badMovies)
+}
+movie()
+
+
+
+//Async Function//
+
+async function afterSetTime(resolveAfter){
+    const promise = new Promise((resolve,reject) => {
```
Comment (bashirmja) [inline javascript/javascript3/week2/index.js:19]: It is hard to find any usage for reject here

File: javascript/javascript3/week2/promise-visual/main.css [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  padding: 0;
+}
+ul.marks li {
+  width: 100px;
+  height: 100px;
+  position: absolute;
+  transition: transform 2s;
+  border-radius: 100%;
+}
+ul.marks li:nth-child(1) {
```
Comment (bashirmja) [inline javascript/javascript3/week2/promise-visual/main.css:27]: It is recommended to use a class selector instead of using nested child selector

Comment (bashirmja) [reply javascript/javascript3/week2/promise-visual/main.css:27]: Please refactor your code and remove all nested child selector in css and js files

**Review by bashirmja — CHANGES_REQUESTED** [review]
Great job, but I made a few comments, and please refactor your code and remove all child selectors as I mentioned in the comments.

File: javascript/javascript3/week2/index.js [hunk 4]
```diff
@@ -0,0 +1,91 @@
+// Movies API//
+async function movie() {
+        const movies = await fetch(
+            "https://gist.githubusercontent.com/pankaj28843/08f397fcea7c760a99206bcb0ae8d0a4/raw/02d8bc9ec9a73e463b13c44df77a87255def5ab9/movies.json"
+        );
+
+    const moviesArray = await movies.json()
+    const badMovies=moviesArray.filter(element => element.rating < 6 && element.year > 2000)
+    .map(element => element);
```
Comment (aina21) [inline javascript/javascript3/week2/index.js:9]: filter result is an array, so I don't think you need a map here.

File: javascript/javascript3/week2/index.js [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+    .then((data) => {
+    console.log(data);
+    });
+}
+setTimeoutPromise(3)
+
+
+///GeoLocation////
+function getCurrentLocation(){
+    let geoLocation = new Promise((resolve, reject) =>{
```
Comment (aina21) [inline javascript/javascript3/week2/index.js:49]: you can use const instead of let.

File: javascript/javascript3/week2/index.js [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+  })
+}
+promiseThen()
+
+////////////////////
+
+async function asyncAwait(){
+    const response = await fetch('https://randomfox.ca/floof/');
+    const data = await response.json()
+    let promise = await new Promise((resolve) => {
```
Comment (aina21) [inline javascript/javascript3/week2/index.js:85]: same here, try to use const until it doesn't let you.

**Review by aina21 — APPROVED** [review]
good job :+1:

**Pull request merged**